spec/features/user_views_booking_activities_spec.rb

```diff
@@ -89,8 +89,8 @@ def when_somebody_else_adds_an_activity
   end
 
   def then_it_appears_dynamically
-    @page.activity_feed.wait_for_dynamically_loaded_activities(3)
-    activity = @page.activity_feed.dynamically_loaded_activities.first
+    expect(@page.activity_feed).to have_activities(count: 5, wait: 10)
```

Comment:
I'd be curious to figure out this issue with needed to specify the wait time explicitly. Not a blocker for now though.